vowpalwabbit/api_status.h

```diff
@@ -0,0 +1,66 @@
+// Copyright (c) by respective owners including Yahoo!, Microsoft, and
+// individual contributors. All rights reserved. Released under a BSD (revised)
+// license as described in the file LICENSE.
+
+#pragma once
+#include <string>
+namespace VW
+{
+  /**
+   * @brief Report status of all API calls
```

Comment:
what constitutes an API call? 

also the class name might be a misnomer, it seems to be a helper for error codes

Reply:
moved to experimental by Jack as discussed